.travis.yml

```diff
@@ -18,8 +18,11 @@ android:
     - 'google-gdk-license-.+'
 
 script:
-  - ./gradlew ktlint test install --info
+  - ./gradlew :plugin:test ktlint install --info
```

Comment:
Why did restrict the tests executinos to the ``:plugin:`` module? This would avoid the tests to be executed for future modules you can create.

Reply:
yep this is for the problem you mention below 😓